conf/application.conf

```diff
@@ -27,7 +27,7 @@ play.filters.csp {
     style-src   = "'self' 'unsafe-inline' https://www.googletagmanager.com https://tagmanager.google.com https://fonts.googleapis.com"
     img-src     = "'self' https://ssl.gstatic.com www.gstatic.com https://www.google-analytics.com data: https://region1.google-analytics.com https://region1.analytics.google.com https://*.google-analytics.com https://*.analytics.google.com https://fonts.gstatic.com https://www.googletagmanager.com"
     font-src    = "'self' https://ssl.gstatic.com www.gstatic.com https://fonts.gstatic.com https://fonts.googleapis.com"
-    connect-src = "https://region1.google-analytics.com https://region1.analytics.google.com https://*.google-analytics.com https://*.analytics.google.com"
+    connect-src = "* https://region1.google-analytics.com https://region1.analytics.google.com https://*.google-analytics.com https://*.analytics.google.com"
```

Comment:
should we use this or use directly psr-backend url?

Reply:
good shout, changed

Reply:
oh it wasn't for backend, I was adding some debug info to pages and it was making a call to github which violates the policy, added it explicitly and removed it in envs here https://github.com/hmrc/app-config-base/pull/9208